PhotoEnhancer/PhotoEnhancer/Filters/InvertFilter.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PhotoEnhancer
+{
+    public class InvertFilter : PixelFilter
+    {
+        public InvertFilter() : base(new InvertParameters()) { }
```

Comment:
Фильтр инверсии не использует параметры. У нас уже имеется класс параметров для такого случая `EmptyPatameters`. Зачем дублировать и создавать другой класс с тем же назначением?
```suggestion
        public InvertFilter() : base(new EmptyPatameters()) { }
```